writing/privacy.md

```diff
@@ -0,0 +1,21 @@
+# Give up all your data
+
+Imagine a world where every advertisement you see is only something that you want to buy and can afford. If you don't want to buy anything, you don't get any ads. When something you don't know about but would love is released, you got told about it. No more spam! Sounds great!
+
+And companies want this too. They waste so much money advertising to people who will never buy their products. They could save so much and spend more on making better inventions for you.
+
+Now imagine that when companies are designing these products, they know exactly how to build the perfect contraption just for you. They could see how you used previous devices, competitors appliances and how to fit their new gizmo into your life. Now you're happy that you have a better product, and they're happy you keep buying from them.
+
+We can get there! But the companies need your data. Ideally all of it. They need to know that you don't like x so they don't waste money advertising it to you. They need to know that you do x, so they can build a y that does z.
+
+At the moment many of us are overly paranoid when it comes to given our personal data to companies. You know no human is looking at your individual cloud data, right? It's not like Jim in data processing is having a gander at 1 users data out of millions and going "Oh the cheeky bastard is searching for scratch and sniff porn mags again". And even if he was, who gives a shit about Jim in data processing. It's only machines and algorithms going through it. Giving trend data to employees so they can help build better products for you, and let you know about things you want.
```

Comment:
*giving our personal data
gander at 1 *user's data
I'd go with "It's only machines and algorithms going through your data, not actual humans.  The humans just see trend data..."